WebCompiler/Helpers/FileFolderHelpers.cs

```diff
@@ -6,21 +7,16 @@ namespace WebCompiler.Helpers
 {
     public static class FileFolderHelpers
     {
-        public static IEnumerable<string> Recurse(string directory)
+        public static IEnumerable<string> Recurse(string directory, List<string> ignoreGlobs)
         {
-            foreach (var file in Directory.GetFiles(directory))
-            {
-                if (!Path.GetFileName(file).StartsWith('_'))
-                {
-                    yield return file;
-                }
-            }
-            foreach (var subdir in Directory.GetDirectories(directory))
+            ignoreGlobs.Add("**/_*.*"); // Also Ignore all files that start with _
```

Comment:
Related bug: https://github.com/excubo-ag/WebCompiler/issues/54

If you see an easy way to support this as well, please consider!

Reply:
This is a good question. A breaking change could be that we remove this explicit rule in the code and suggest a default template in the README file with this explicitly noted?

**Example**
```json
  "CompilerSettings": {
     "Ignore": [ "bin/**/*", "obj/**/*", "wwwroot/*.scss", "wwwroot/css/specific-file.scss", "wwwroot/_lib/**/*.scss", "**/_*.*" ],
  }
```

Reply:
What happens if we set the default `Ignore` value to `new List<string> { ""**/_*.*" }`? Shouldn't that keep the behavior as-is and make it easier to opt out?

Reply:
Done 👍